src/hotspot/share/gc/shared/workgroup.hpp

```diff
@@ -329,7 +333,18 @@ class SubTasksDone: public CHeapObj<mtInternal> {
   // cleared.)
   //
   // n_threads - Number of threads executing the sub-tasks.
-  void all_tasks_completed(uint n_threads);
+  // followed by vararg skipped tasks
```

Comment:
The description comment should be augmented to describe the optional skipped values.